browser/ui/webui/brave_new_tab_message_handler.cc

```diff
@@ -29,7 +29,8 @@
 
 using ntp_background_images::features::kBraveNTPBrandedWallpaper;
 using ntp_background_images::prefs::kNewTabPageShowBackgroundImage;
-using ntp_background_images::prefs::kNewTabPageShowBrandedBackgroundImage;
+using ntp_background_images::prefs::kNewTabPageShowSponsoredImagesBackgroundImage;  // NOLINT
```

Comment:
this doesn't seem right to me, we're going to make people opt out of these individually? Shouldn't there just be one pref for all branded background images?

Reply:
Per the spec, person would need to go to settings to turn off this "theme". 
https://docs.google.com/document/d/1DlCHfQXZS3MamKnjCkEHC7yMyJfNObKlgE9X-tJhuEc/edit

The line below is new (above is a just a rename). Having a single boolean preference would work for now - but in the future, `kNewTabPageShowSuperReferrerBackgroundImage` might be best as a string. This would allow it to be empty or one of multiple values ("themes")
cc: @anthonypkeane, @rebron

Reply:
One another reason about using new prefs is super referrer can be back to normal referrer.
Then, browser will install sponsored images.
and opposite way is also possible. normal referrer can become super referrer.

Reply:
@simonhong the preferences should not be used to select between sponsored and referral images. The themes thing doesn't make any sense to me and I commented here https://docs.google.com/document/d/1DlCHfQXZS3MamKnjCkEHC7yMyJfNObKlgE9X-tJhuEc/edit?disco=AAAAGWQgyPA

Reply:
it seems especially problematic as a string if the theme goes away like I think it does according to the spec

Reply:
@bridiver `kNewTabPageShowSponsoredImagesBackgroundImage` will be used only for turn on/off the sponsored images. If super referral is in active, it's not used.
and newly introduced `kNewTabPageSuperReferralThemesOption` will be used while super referral is in active. It is used to switch between normal background and super referral.

and according to your question in spec, themes sub page will not be visible when super referral campaign ends.